cmd/account/mgmt/account-assign_test.go

```diff
@@ -130,6 +147,43 @@ func TestFindUntaggedAccount(t *testing.T) {
 	}
 }
 
+func TestCreateAccount(t *testing.T) {
+	mocks := setupDefaultMocks(t, []runtime.Object{})
+
+	mockAWSClient := mock.NewMockClient(mocks.mockCtrl)
+
+	seed := int64(1)
+
+	randStr := RandomString(6)
+	accountName := "osd-creds-mgmt" + "+" + randStr
+	email := accountName + "@redhat.com"
+
+	createId := "car-random1234"
+
+	mockAWSClient.EXPECT().CreateAccount(&organizations.CreateAccountInput{
+		AccountName: &accountName,
+		Email:       &email,
+	}).Return(&organizations.CreateAccountOutput{
+		CreateAccountStatus: &organizations.CreateAccountStatus{Id: &createId},
+	}, nil)
+
+	awsDescribeOutput := &organizations.DescribeCreateAccountStatusOutput{}
```

Comment:
Problem is here - this is returned from the create call, then we try access it here
https://github.com/openshift/osdctl/pull/133/files#diff-1b2f7e3efa0d33561aa4d30779d7dea340714149fd3233ca89527609670aac5dR273